### PR TITLE
Improve xCAT uninstall documentation

### PR DESCRIPTION
**Pull request by gurevichmark**
Improve xCAT uninstall documentation

### CODE REVIEW

File: docs/source/guides/install-guides/maintenance/uninstall_xcat.rst [hunk 1]
```diff
@@ -48,128 +48,18 @@ Remove xCAT Files
 
 1. Remove the xCAT RPMs
 
-  There is no easy way to distinct all the packages depending by xCAT. For packages shipped by xCAT, you can remove them by the commands below.
+  There is no easy way to identify all xCAT packages. For packages shipped by xCAT, you can remove them by using the commands below.
   
   [RHEL and SLES] ::
 
-      rpm -qa |grep -i xcat
+      yum remove '*xcat*'
```
Comment (whowutwut) [inline docs/source/guides/install-guides/maintenance/uninstall_xcat.rst]: Does this catch all of them, do we need .... `yum remove '*xcat*' '*xCAT*'`  ?

File: docs/source/guides/install-guides/maintenance/uninstall_xcat.rst [hunk 2]
```diff
@@ -178,31 +68,35 @@ Remove xCAT Files
...
+  
+        http://xcat.org/files/xcat/repos/yum/xcat-dep/<os>/<arch>
+
+    [Ubuntu] ::	
+  
+        http://xcat.org/files/xcat/repos/apt/xcat-dep/pool/main
+
 
-  Generally, we use ``yum install xCAT`` to install xCAT, so these are some RPMs shipped by operating system are installed during xCAT installation. We don't have an easy way to find out all of them, but keep these RPMs are harmless. 
+  When ``yum install xCAT`` is used to install xCAT, some RPMs shipped by operating system are installed. There is no easy way to find all of them, but keeping those RPMs is harmless.
```
Comment (whowutwut) [inline docs/source/guides/install-guides/maintenance/uninstall_xcat.rst]: Suggest: 

```
When ``yum install xCAT`` is used to install xCAT, dependency RPMs provided by the Operating System will be installed.  Keeping those rpms installed on the system is harmless. 
```

File: docs/source/guides/install-guides/maintenance/uninstall_xcat.rst [hunk 3]
```diff
@@ -178,31 +68,35 @@ Remove xCAT Files
...
-  By default, xCAT use SQLite, remove SQLite data file under ``/etc/xcat/``. ::
+  By default, xCAT uses SQLite, remove SQLite data files under ``/etc/xcat/``. ::
 
     rm -rf /etc/xcat
 
-4. Remove xCAT related file(Optional)
+4. Remove xCAT related files (optional)
 
-  XCAT has ever operated below directory when it was running. Do judgment by yourself before removing these directory, to avoid removing some directories used for other purpose in your environment. ::
+  XCAT might have also created additional files and directories below. Be careful removing these files, as they might be used for other purposes in your environment. ::
```
Comment (whowutwut) [inline docs/source/guides/install-guides/maintenance/uninstall_xcat.rst]: Suggest:  `Be careful removing these files... ` -> `Take caution when removing these files as they may be used for other purposed in your environment.`

File: docs/source/guides/install-guides/maintenance/uninstall_xcat.rst [hunk 4]
```diff
@@ -48,128 +48,18 @@ Remove xCAT Files
...
+      yum remove '*xcat*'
 
   [Ubuntu] ::	  
   
       dpkg -l | awk '/xcat/ { print $2 }'
 
-  If you want to remove more cleanly, the list below maybe helpful. Listed are the packages of xcat installation tarball. Some RPMs may not to be installed in a specific environment.
+  To do an even more thorough cleanup, use links below to get a list of RPMs installed by xCAT. Some RPMs may not to be installed in a specific environment.
 
-  * XCAT Core Packages list (xcat-core):
```
Comment (whowutwut) [inline docs/source/guides/install-guides/maintenance/uninstall_xcat.rst]: 👍  Nice remove to simplify the list!

File: docs/source/guides/install-guides/maintenance/uninstall_xcat.rst (hunk 1 above)
Comment (gurevichmark) [reply docs/source/guides/install-guides/maintenance/uninstall_xcat.rst]: Good point. Both need to be executed. I could not find "ignore case" option for `yum`.

**Pull request merged**